frontend/src/services/list.service.ts

```diff
@@ -29,7 +29,7 @@ const ListService = {
         id: id
       }]
     );
-    const ogData = <IOGData>response.data;
+    const ogData = <IOGData>response.data[0];

```

Comment:
If you url bundle has multiple links, does this still work?  I know @michaeldeongreen change the response to an array to support cases where multiple link validates happen at the same time.  I'm assuming the frontend makes a call per link??

Reply:
yes it looks like the frontend makes a call per link.

Reply:
Tested, it worked with multiple links because the frontend always makes a call per link.